en/docs/develop-components/develop-services/develop-mock-server.md

```diff
@@ -0,0 +1,73 @@
+# Build and Deploy a Mock Server
```

Comment:
Shall we use "Expose a Mock Server Using the Open API Specification"  here?
Suggesting because we have used a similar topic name in the mkdocs.yml file as the left nav title for this page. The name has to be consistent in both places and the markdown file name also should be the same. See https://github.com/wso2/docs-choreo-dev/wiki/Markdown-File-Naming-Conventions-and-Content-Formatting#file-naming-convention